packages/ckeditor5-list/tests/listediting.js

```diff
@@ -291,6 +291,21 @@ describe( 'ListEditing', () => {
 
 			sinon.assert.calledWithExactly( editor.execute, 'outdentList' );
 		} );
+
+		it( 'should outdent empty list when list is nested in block quote', () => {
+			const domEvtDataStub = { preventDefault() {}, direction: 'backward' };
+
+			sinon.spy( editor, 'execute' );
+
+			setModelData(
+				model,
+				'<paragraph>x</paragraph><blockQuote><listItem listType="bulleted" listIndent="0">[]</listItem></blockQuote>'
+			);
+
+			editor.editing.view.document.fire( 'delete', domEvtDataStub );
+
+			sinon.assert.calledWithExactly( editor.execute, 'outdentList' );
```

Comment:
I know you were consistent with all the other tests here, but just to clarify – I think all of them should be checking the actual content of the model too, not only whether the command was executed. We don't know if some other handler didn't react or the command didn't do something unexpected (quite probable when changing its behavior).